src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,19 @@
+import { deleteComment } from "../utils/api";
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
+
```

Comment:
I'm absolutely happy for you to keep your DeleteComment as a separate component. However, as there isn't too much functionality or too many elements in the return it would also be fine to keep the delete stuffs in the Comments component. 

It's definitely a fine line balancing when to extract into a further component and something that will come with practice